src/ensembl/package.json

```diff
@@ -45,6 +45,7 @@
     "dotenv": "10.0.0",
     "ensembl-genome-browser": "https://raw.githubusercontent.com/Ensembl/ensembl-genome-browser-assets/master/assets-80f51620ed443c640cdfd6b5aebd505b.tar.gz",
     "express": "4.17.1",
+    "filesize": "7.0.0",
```

Comment:
This is a small library for formatting file size into a human-readable string. Looks more robust than what I would have written myself.